client-go/controller/api/certs.go

```diff
@@ -0,0 +1,26 @@
+package api
+
+// Cert is the definition of the cert object.
+type Cert struct {
+	Updated string `json:"updated,omitempty"`
+	Created string `json:"created,omitempty"`
+	Name    string `json:"common_name"`
+	Expires string `json:"expires"`
+	Owner   string `json:"owner,omitempty"`
```

Comment:
are these `omitempty` fields here strictly to workaround tests or something? Certs should always have an owner.